### PR TITLE
Vet Clinic: Query and update db

### PR DESCRIPTION
**Pull request by kkolade**
# In this PR:
- Add a column species of type string to the animals table. 
- Insert more animals into the animals table.
- In a transaction, I update the animals table by setting the species column to unspecified. Verify the change made. Then roll back the change and verify that the species columns went back to the state before the transaction.
- Inside a transaction:
   - I update the animals table by setting the species column to digimon for all animals 
   - that have a name ending in mon.
   - I update the animals table by setting the species column to pokemon for all animals that don't have species already set.
   - I commit the transaction.
   - I verify that the changes were made and persists after commit.
- Inside a transaction  I delete all records in the animals table, then roll back the transaction.
- After the rollback  I verify if all records in the animals table still exists. 
- Inside a transaction:
   - I delete all animals born after Jan 1st, 2022.
   - I create a savepoint for the transaction.
   - I update all animals' weight to be their weight multiplied by -1.
   - I rollback to the savepoint
   - I update all animals' weights that are negative to be their weight multiplied by -1.
   - I commit transaction
- I write queries to answer the following questions:
   - How many animals are there?
   - How many animals have never tried to escape?
   - What is the average weight of animals?
   - Who escapes the most, neutered or not neutered animals?
   - What is the minimum and maximum weight of each type of animal?
   - What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Screen Shot 2023-05-03 at 23 33 25

Screen Shot 2023-05-04 at 22 52 01
"https://user-images.githubusercontent.com/25391226/236349105-2a848ea5-e8fb-4e2c-85d8-8d73e75f495e.png">
eff
Screen Shot 2023-05-04 at 21 15 06

Screen Shot 2023-05-04 at 22 20 17
c="https://user-images.githubusercontent.com/25391226/236349066-75fccc25-a9a9-45d9-8157-11c645838690.png">
f-49cc-91fa-c6836f7a45ed.png">
Screen Shot 2023-05-04 at 21 01 35

Screen Shot 2023-05-04 at 20 19 13

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
          ('Gabumon', '2018-11-15', 2, '1', 8),
          ('Pikachu', '2021-01-07', 1, '0', 15.04), 
          ('Devimon', '2017-05-12', 5, '1', 11);
 
 SELECT * FROM animals;  -- to verify inserted data
+
+-- Add a new column to the table
+ALTER TABLE animals
+ADD COLUMN species VARCHAR(80);
+
```
Comment (Owusu-Desmond) [inline data.sql]: - [ ] Please ensure that everything that creates or changes the table structure should be saved in the `schema.sql` file. :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @kkolade , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (kkolade) [reply data.sql]: This issue is fixed now

**Review by Owusu-Desmond — APPROVED** [review]
Hi @kkolade , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Descriptive pull request
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**